src/Microsoft.DotNet.Interactive.Tests/Connection/SerializationTests.cs

```diff
@@ -233,23 +235,22 @@ public static IEnumerable<object[]> Events()
 
             IEnumerable<KernelEvent> events()
             {
-                var submitCode = new SubmitCode("123");
 
                 yield return new CodeSubmissionReceived(
-                    submitCode);
```

Comment:
Any reason you inlined this?

Reply:
because reusing same command over and over is making problems with the routing slip

Reply:
Interesting. That's probably a good thing. What kind of problem?